sentry-core/src/scope/real.rs

```diff
@@ -8,12 +8,24 @@ use crate::protocol::{Attachment, Breadcrumb, Context, Event, Level, User, Value
 use crate::session::Session;
 use crate::Client;
 
+#[non_exhaustive]
+pub enum ScopeUpdate {
```

Comment:
at least this type should live in the public API, independent of the `client` feature.